blocks/section/section.css

```diff
@@ -0,0 +1,6 @@
+.section{
+  display: flex;
+  align-items: center;
```

Comment:
@naumch1k I checked to see if it would affect the correct look and I didn't like the result, or did you mean to change the way the text is set to center in the parent block?

Reply:
@rasokolovska Could you please provide a screenshot showing how your element looks without **`align-items: center;`**? Also, make sure **to remove height: `240px;`** in the screenshot, as I don't think we need that property at all.